frontend/src/container/AppLayout/index.tsx

```diff
@@ -1,44 +1,23 @@
-import ROUTES from 'constants/routes';
 import TopNav from 'container/Header';
 import SideNav from 'container/SideNav';
-import history from 'lib/history';
-import React, { ReactNode, useEffect, useState } from 'react';
+import React, { ReactNode } from 'react';
 import { useSelector } from 'react-redux';
 import { AppState } from 'store/reducers';
 import AppReducer from 'types/reducer/app';
-
 import Feedback from './FeedBack';
-import { Content, Footer, Layout } from './styles';
+import { Content, Layout } from './styles';
 
 const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
 	const { isLoggedIn } = useSelector<AppState, AppReducer>((state) => state.app);
 
-	const [isSignUpPage, setIsSignUpPage] = useState(
-		ROUTES.SIGN_UP === location.pathname,
```

Comment:
why we are removing this one ?

Reply:
basically what's going on is, if its signup page, and user is logged in, we set `isSignUpPage` to false and then we use that value to show/hide components. 
That effectively translates to - if user is logged in, show `SideNav` and `TopNav`, else hide both, regardless of what page that is.
So that was misleading and causes confusion. 

One thing that just came to my attention, I also removed the logic to navigate to sign up page if user is not logged in. Thanks a lot for pointing this out! Fixing this